.github/workflows/lint.yml

```diff
@@ -8,7 +8,7 @@ run-name: Executing lint QA on ${{ github.repository }} 🚀
 
 on:
   pull_request:
-    branches: [main]
+    branches: [main, main-application-*]
```

Comment:
Not a blocker for this PR, wondering if we should move away from `main-application-*` since it's "all application" now. Maybe could be e.g `main-no-pdf`, `main-file-upload` etc

Reply:
Updated.